DataAccessLayer/SQLScripts/LanguageSkillsDB_postgres_create_table.sql

```diff
@@ -122,3 +134,5 @@ ALTER TABLE "WordTranslations" ADD CONSTRAINT "WordTranslations_fk1" FOREIGN KEY
 ALTER TABLE "TestTranslations" ADD CONSTRAINT "TestTranslations_fk0" FOREIGN KEY ("TestId") REFERENCES "Tests"("Id");
 ALTER TABLE "TestTranslations" ADD CONSTRAINT "TestTranslations_fk1" FOREIGN KEY ("LanguageId") REFERENCES "Languages"("Id");
 
+ALTER TABLE "LanguageTranslations" ADD CONSTRAINT "LanguageTranslations_fk0" FOREIGN KEY ("LanguageWordId") REFERENCES "Languages"("Id");
```

Comment:
bad naming convention for foreign keys - please give more meaningful names to foreing keys